wiii-desktop/src/styles/globals.css

```
@@ -2866,36 +2956,106 @@ html.dark .shiki span {
+  grid-template-columns: max-content minmax(0, 1fr);
+  gap: 0.52rem;
+  margin-top: 0.48rem;
+  padding: 0.48rem 0.56rem;
+  border: 1px solid color-mix(in srgb, var(--border) 54%, transparent);
+  border-radius: 6px;
+  background: color-mix(in srgb, var(--surface) 54%, transparent);
+  color: color-mix(in srgb, var(--text) 78%, var(--text-tertiary));
   font-size: 0.76rem;
-  line-height: 1.6;
+  line-height: 1.55;
   word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)